perceptual_filter/InappropriateBehavior inappropriate_behavior

uint32 person_index

geometry_msgs/Point head_position
